core/runtime/runtime_api/impl/grandpa_api.hpp

```diff
@@ -29,14 +29,11 @@ namespace kagome::runtime {
     outcome::result<AuthorityList> authorities(
         const primitives::BlockId &block_id) override;
 
-    outcome::result<primitives::AuthoritySetId> current_set_id(
-        const primitives::BlockHash &block) override;
-
    private:
     std::shared_ptr<blockchain::BlockHeaderRepository> block_header_repo_;
     std::shared_ptr<Executor> executor_;
   };
 
 }  // namespace kagome::runtime
 
-#endif  // KAGOME_RUNTIME_GRANDPAAPI
```

Comment:
clang-format was reverted

Reply:
If this clang-format happened in the reverted commit, it will be reverted. I will bring it back after I fix it, so I don't think it's worth to pick parts of it which are not to be reverted now.